### PR TITLE
main.coffee: Updates coala's min version to 0.10.0

### PR DESCRIPTION
**Pull request by EverWinter23**
update COALA_MIN_VERSION from '0.9.1' to '0.10.0'

### CODE REVIEW

**Comment by alisianoi** [comment]
Hey, @EverWinter23 , you are not following [commit message guidelines][1]

In particular, please use imperative: `Updated` -> `Update`

Also, circleci appears to be failing because of some unrelated issue.

[1]: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by EverWinter23** [comment]
Should i keep working on it?

**Comment by RohanVB** [comment]
Try to figure out what's causing circleci to fail on this build. 
When I pushed the same PR, ci passed. 
(https://github.com/coala/coala-atom/pull/44)

Also, fix the commit message. See: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by RohanVB** [comment]
@EverWinter23 
Good job, it seemed to be a dependency issue. Can you please squash the extra commits? 
Also, `Updates` -> `Update` in the shortlog. 

You can also remove the indent tab in the commit body and have a new line between the issue reference and commit body so it complies to: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

Other than that, lgtm :)

**Comment by EverWinter23** [comment]
Finally Done!!!

File: circle.yml [hunk 1]
```diff
@@ -3,17 +3,20 @@ dependencies:
...
     - curl -L -o atom.deb https://atom.io/download/deb
     - sudo dpkg --install atom.deb
-
+    - sudo apt-get -f install
     # Set python version
     - pyenv local 3.4.3
 
     # Install coala
+    - pip install --upgrade pip
+    - pip install six
```
Comment (RohanVB) [inline circle.yml]: Is this required? Didn't your last build pass without `six`?
Also, the current passing build on master does work without upgrading pip, with v 6.x or something. Could you check if these 2 lines are required again please?

Comment (EverWinter23) [reply circle.yml]: i got six module not defined

Comment (EverWinter23) [reply circle.yml]: i'll try without that again

Comment (EverWinter23) [reply circle.yml]: cuz last time i hand't upgraded pip

Comment (RohanVB) [reply circle.yml]: Sounds good. Better to spend an extra few minutes to verify :)

Comment (EverWinter23) [reply circle.yml]: Made changes.

**Comment by EverWinter23** [comment]
Just upgrading pip works.

**Comment by RohanVB** [comment]
ack 617d114141e1640e99744df8949ce69469226aac

**Comment by meetmangukiya** [comment]
@RohanVB any reason this is not merged ?

**Comment by RohanVB** [comment]
@meetmangukiya  [March 20, 2017 4:23 PM](https://gitter.im/coala/coala?at=58cfb4aff7f7d4810436f417)
(tldr; only members of the coala maintainers' group are allowed to merge)

**Pull request merged**